### DISCUSSION TITLE
Img doesnt generate

### DISCUSSION THREAD

**Discussion by Deforal** [lllyasviel/Fooocus#2802] [Q&A] [answered]
During generation my loading stops at very start and there is no progress, I did system swap. Here how it looks:
![image](https://github.com/lllyasviel/Fooocus/assets/129059957/fc4d9784-40b9-44c4-98ee-3b1e63a98dcb)
My graphics car is NVIDIA Geforce GTX 1650 with general memory(?) of 8 gb and video memory of 4 gb. GPU Intel(R) Core(TM) i3-9100F CPU @ 3.60GHz   3.60 GHz. RAM is 16GB. Dont know what is the issue

**Comment by Rogala** [comment]
Install the latest drivers from Nvidia and make the swap file larger than 40GB

**Reply by Deforal** [reply]
So I did as you said and at first it actually worked but loading was 3+ minutes so I closed the window. Second time error appeared again, maybe because I increased swap file amount after first time. For third time happened the same thing.
![image](https://github.com/lllyasviel/Fooocus/assets/129059957/c07d227e-b8f1-446a-8b2c-23cb74087d1d)
Its in russian, sorry about that. Is this how it supposed to look like?

**Reply by Rogala** [reply]
Show what happens in the console, after running Fooocus and after running picture creation

**Reply by Deforal** [reply]
I gave it some time:
![image](https://github.com/lllyasviel/Fooocus/assets/129059957/784dc761-df5d-45a6-a2a4-30f88a294833)

**Reply by ILikeToasters** [reply]
Do you have multiple instances of Fooocus running? Looking at  the port number tells me it didn't use the default of 7865 and is using 7867 which tells me you have something else using 7865 and 7866 which I think means you probably have multiple CMD windows open with it running.

**Reply by Deforal** [reply]
No, I started it first thing after my pc turned on, I opened  run.bat and nothing else. Or maybe CMD windows open on their own, not sure. When I will get home I will double check.

**Reply by Deforal** [reply]
So, I tried booting it again, now it is 7865, but the initial issue arised again.

**Reply by Rogala** [reply]
You have run Image Generation, show me what is happening in the console. And in the Task Manager. You can show a screenshot.

**Reply by Deforal** [reply]
![image](https://github.com/lllyasviel/Fooocus/assets/129059957/e612f9da-1b59-4ab6-9532-7b3967b9bada)

**Reply by Rogala** [reply]
Skin the console after running the image generation, we have already seen that in the console after running the botnik, and Task Manager show the CPU, Memory and GPU performance tab during the image generation.

**Reply by Deforal** [reply]
https://github.com/lllyasviel/Fooocus/assets/129059957/89275479-fa7a-4b0b-81fb-0059b6834eaf

**Reply by Deforal** [reply]
Is this what you wanted? What other console did you want me to use? And also there is no need for russian, I can understand you just fine, it may create misunderstanding.

**Reply by Rogala** [reply, marked as answer]
We don't understand each other, I can't help you anymore.

You're running on a slow disk drive, this process takes a long time.
